unit_tests/tests/sensor/test_sensor_init.cpp

```diff
@@ -131,6 +131,7 @@ TEST(SensorInit, DriverIntentNoPedal) {
 
 	initTps();
 
+	ASSERT_TRUE(Sensor::hasSensor(SensorType::AnalogTps1));
```

Comment:
this doesn't need a different sensor type, does it?

Reply:
won't the SENT sensor just register itself?

Reply:
sorry do not follow. not sure if https://github.com/rusefi/rusefi/pull/5114/commits/1d4e41bd2853ac09105138c1ce36a3235800084a answers any questions

Reply:
imo this is a misuse of the proxy sensor, since analog TPS and SENT tps are mutually exclusive, there is no reason to process both as separate sensors instead of registering one vs other

Reply:
thank you